hudi-hive/src/test/java/org/apache/hudi/hive/TestUtil.java

```diff
@@ -107,7 +107,7 @@ public static void setUp() throws IOException, InterruptedException, URISyntaxEx
     hiveSyncConfig.hivePass = "";
     hiveSyncConfig.databaseName = "testdb";
     hiveSyncConfig.tableName = "test1";
-    hiveSyncConfig.basePath = "/tmp/hdfs/HiveSyncToolTest/";
+    hiveSyncConfig.basePath = "/tmp/hdfs/TestHiveSyncTool/";
```

Comment:
Except for renaming, it is the only change. It should not affect the failed test case. WDYT? @vinothchandar

Reply:
No this should not affect anything.. Can you reproduce the error locally?  the VM timeout happened in the past from leaking writeClient (i.e tests not calling writeClient.close()). But we fixed it and its all been stable since june. Again does not seem related to this,

Reply:
I ran this test several times. It can be passed (although the test took much time).